R/RunStep.R

```diff
@@ -55,10 +55,15 @@
 #' @export
 
 RunStep <- function(lStep, lData, lMeta, lSpec = NULL) {
+  log4r::info(.package_logger, "[RunStep] Starting function execution.")
   # prepare parameter list inputs
   params <- lStep$params
 
   cli::cli_h3("Evaluating {length(params)} parameter(s) for {.fn {lStep$name}}")
+  log4r::info(
```

Comment:
It also seems like we can write a function that will run the cli and log4r to get the console message and the log entry, could clean up our cli usage to:

cli::cli_inform ~= log4r::info
cli::cli_warn ~= log4r::warn
cli::cli_abort ~= log4r::error